packages/entities/entities-shared/src/types/entity-base-config-card.ts

```diff
@@ -1,5 +1,23 @@
 import type { KonnectConfig, KongManagerConfig } from './index'
 
+export enum SupportedEntityType {
+  CaCertificate = 'ca_certificate',
+  Certificate = 'certificate',
+  Consumer = 'consumer',
+  ConsumerGroup = 'consumer_group',
```

Comment:
@kaiarrowood We need an entry for `ConsumerCredential` to display Terraform since we do display JSON/YAML Configuration in the ConsumerCredentialForm
```suggestion
  ConsumerCredential = 'consumer_credential',
```
<img width="1728" alt="Screenshot 2024-08-13 at 1 55 59 PM" src="https://github.com/user-attachments/assets/3e4ffdfa-6a72-422c-83fe-b1c3ebe9247c">

Reply:
Consumer credentials are plugins 🤔 I need to look into this

Reply:
Will fix in a future update